glide.yaml

```diff
@@ -1,19 +1,26 @@
 package: github.com/openshift/openshift-azure
 import:
 - package: github.com/openshift/api
-  version: release-3.10
```

Comment:
This was an issue with MSFT side of vendor. Problems with ServiceCatalog dep clash. Need to confirm with  MSFT before merging. Or maybe move to separate PR and send it to MSFT to check.